package.json

```diff
@@ -36,7 +36,7 @@
     "decimal.js": "^10.2.1",
     "formik": "^2.2.9",
     "google-protobuf": "^3.15.8",
-    "ldk": "^0.5.2",
+    "ldk": "^0.5.3",
     "lodash.debounce": "^4.0.8",
     "lottie-web": "^5.7.8",
     "marina-provider": "^1.7.0",
```

Comment:
Should it be 1.7.1?

Reply:
Yep! It must be to have new interface

Reply:
https://github.com/vulpemventures/marina-provider/commit/48c705d3e81eec01551d487d4461783ccec78aa0